content/en/docs/architecture/ci-operator.md

```diff
@@ -362,6 +362,33 @@ releases:
         upper: "4.5.0-0"
 {{< / highlight >}}
 
+### Testing With a Cluster from a Cluster Pool
+Instead of using `releases` stanza in the configuration, a job can *claim* a cluster from a cluster pool and use it for testing _"something on OpenShift"_.
```

Comment:
We should also describe what's the difference: instead of installing a cluster from scratch on-demand (takes time but allows clsuter customization and is necessary when you test OCP) you can claim a pre-installed one (faster, but you can only use this whne you need sufficiently "generic" cluster, and you cannot test OCP  components with tthis)

Reply:
There is also the "Should I Use an Ephemeral or Published Release?" section, which should be extended with infromation about claimed clusters.

Reply:
Document how to do this, not how to *not* do it - don't talk about other features like `releases` unless you _must_.